app/views/shift_surveys/_form_fields.html.haml

```diff
@@ -0,0 +1,76 @@
+  .survey-text-area {
+    display:none;
+  }
+- if f.object.errors.any?
+  #error_explanation
+    %h5
+      = pluralize(f.object.errors.count, "error")
+      prohibited this task from being saved:
+    %ul
+      - f.object.errors.full_messages.each do |msg|
+        %li= msg
+.grid-x
+  .cell
+    = f.label :supplies do
+      Did you have adequate supplies to care for the child(ren) during your time at the office?
+      <br/>
+      .survey-radio
+        = f.radio_button :supplies, true
+        Yes 
+        <br/>
+        = f.radio_button :supplies, false 
+        No
+        = f.text_area :supplies_text, class: "survey-text-area", :rows => 4, placeholder: "Please explain why you answered 'No'"
+    %hr
+    = f.label :response_time do
+      Did the assigned social worker respond to questions or concerns in a timely manner?
+      <br/>
+      .survey-radio
+        = f.radio_button :response_time, true
+        Yes 
+        <br/>
+        = f.radio_button :response_time, false
+        No
+        = f.text_area :response_time_text, class: "survey-text-area", :rows => 4, placeholder: "Please explain why you answered 'No'"
+    %hr
+    = f.label :hours_match do
+      Did the hours you signed up for match the hours you volunteered?
+      <br/>
+      .survey-radio
+        = f.radio_button :hours_match, true
+        Yes 
+        <br/>
+        = f.radio_button :hours_match, false
+        No
+        = f.text_area :hours_match_text, class: "survey-text-area", :rows => 4, placeholder: "Please explain why you answered 'No'"
+    %hr
+      How are you feeling after your shift at the child welfare office? (check all that apply)
+    = f.label :ratings do
+      = f.collection_check_boxes(:ratings, {'Happy': 'Happy', 'Contented': 'Contented', 'Excited': 'Excited', 'Restless': 'Restless', 'Sad': 'Sad', 'Worried': 'Worried', 'Angry': 'Angry'}, :last, :first)
+    = f.label :ratings_text do
+      Would you like to elaborate? (Optional)
+      = f.text_area :ratings_text, :rows => 4, :placeholder => 'Optionally elaborate'
+    %hr
+    = f.label :comments do
+      Your feedback is important to us, and we love hearing from our volunteers! Do you have any comments or kudos you’d like to pass along? (Optional)
+      = f.text_area :comments, :rows => 4, :placeholder => 'Enter comments or kudos here'
+    %hr
+    = f.label :questions do
+      Any questions for your Volunteer Coordinator? (Optional)
+      = f.text_area :questions, :rows => 4, :placeholder => 'Enter notes for your Volunteer Coordinator here'
+
+:javascript
+  $(document).ready( function() {
+    $('input:radio').change(
+      function() {
+        let radio_name_text = $(this).attr('name');
+        radio_name_text = radio_name_text.replace(']', '_text]');
+        if ($(this).val() == 'false') {
+          $(`textarea[name="${radio_name_text}"]`).show();
+        } else {
+          $(`textarea[name="${radio_name_text}"]`).hide();
+        }
+      }
+    );
+  });
```

Comment:
I know you're working on moving this to a stimulus controller. Just commenting for the history books :-)